nsqadmin/static/js/views/app.js

```diff
@@ -136,6 +136,10 @@ var AppView = BaseView.extend({
     },
 
     onLinkClick: function(e) {
+        if (e.ctrlKey || e.metaKey) {
+          // allow ctrl+click to open in a new tab
+          return true
```

Comment:
> I'm not sure i follow. Are you implying an empty `return`?

Yeah, I think it can just be an empty `return`. I believe jQuery only cares if you `return false` - in which case it's the same as calling `stopPropagation` _and_ `preventDefault`. [This](http://stackoverflow.com/questions/4379403/jquery-event-handlers-return-values) might be helpful.

> Why would i want to `stopPropagation`?

I'm not sure why it's being called right now. I'm just saying if there's a good reason to call it, it probably should be called even when ctrl-clicking. I'm assuming it's there for some reason, but I may be wrong. I'm not too familiar with this codebase.